pool/test/user_import_test.ml

```diff
@@ -0,0 +1,159 @@
+open Pool_common.Message
+
+module Data = struct
+  let token = "123123"
+  let password = "Password1!"
+end
+
+let get_or_failwith_pool_error = Test_utils.get_or_failwith_pool_error
+
+let create_user_import ?(token = Data.token) user =
+  let open User_import in
+  let user_uuid =
+    let open Pool_context in
+    match user with
+    | Contact contact ->
+      Contact.(id contact |> Id.value |> Pool_common.Id.of_string)
```

Comment:
Contact.id is still of type `Pool_common.Id`.
The following would be enough:
```suggestion
      Contact.(id contact)
```

When did we plan to update the contact id to its type?